proc_nutation.py

```diff
@@ -1,53 +1,84 @@
+from pylab import *
 from pyspecdata import *
 from scipy.optimize import minimize
-from proc_scripts import hermitian_function_test,zeroth_order_ph,postproc_dict,fl_mod,center_CPMG_echo
+from proc_scripts import hermitian_function_test,zeroth_order_ph,postproc_dict,fl_mod
```

Comment:
is center CPMG echo just calling the hermitian function test?  Make sure that you are using these imports, and delete any not used.